external_review/collate_save_census.py

```diff
@@ -0,0 +1,68 @@
+"""This functionality should eventually be in the same file as external_review_collate 
+where housing production is collated and saved. Up against deadline it's easier to
+write a new file but this step can be DRY'd out and brought down to a simplier format"""
+
+from os import path, makedirs
+import typer
+from aggregate.decennial_census.decennial_census_001020 import decennial_census_001020
+from aggregate.PUMS.pums_2000_demographics import pums_2000_demographics
+from aggregate.PUMS.pums_2000_economics import pums_2000_economics
+from aggregate.PUMS.pums_0812_1519_demographics import acs_pums_demographics
+from aggregate.PUMS.pums_0812_1519_economics import acs_pums_economics
+
+
+from aggregate.load_aggregated import initialize_dataframe_geo_index
+
+app = typer.Typer()
+
+dec_census_year_mapper = {"1519": 2020, "0812": 2010, "2000": 2000}
```

Comment:
I think te made a similar comment but I think it makes sense to stay consistent between year as an int vs string